fvtest/compilertriltest/SimplifierFoldAndTest.cpp

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
```

Comment:
Just realized this licence header is wrong, but I am about to run out the door... hence WIP 👅 

I'll get to this by Monday.